rpa-home/modules/ROOT/pages/_partials/release-notes/rpa-builder-6.7.2-release-notes.adoc

```diff
@@ -0,0 +1,62 @@
+= RPA Builder 6.7.2 Release Notes
+
+RPA Builder is the on-premises application that enables you to build automations by specifying the action steps of the process.
+
+include::partial$release-notes/rn-last-version-message.adoc[] 
+
+== September 27, 2024
+ 
+// == What's New
+include::partial$release-notes/rn-headings.adoc[tag=whatsNew,leveloffset=+1]
+
+//new action steps 
+
+//enhancements to action steps
+* The RPA Builder setup file now includes a digital signature to prevent security warnings. 
```

Comment:
It's not THE setup file, it is a file / application used in the setup which was signed. Also I think it should not be under "enhancements to action steps". It can go under "other updates".
I think we can name the file because the one customer was complaining their security blocked it when installtion, thats why we have now signed it.
Something like "The StNetSh.exe tool in RPA Builder setup now includes a digital signature to prevent security warnings."